code/modules/reagents/reagent_containers.dm

```diff
@@ -98,3 +98,26 @@
 	var/list/mc = ReadRGB(mix_color_from_reagents(reagents.reagent_list))
 	filler.color = RGB_CONTRAST(mc[1], mc[2], mc[3])		// paint in color of drink
 	return filler
+
+/obj/item/weapon/reagent_containers/proc/cause_syringe_fear(mob/living/carbon/human/user) //Quality proc. Because someone didn't insert the cyanide-syringe into the syringe.dm file
```

Comment:
Напиши коммент НАД функцией, а не вот как ты щас сделал. 

А еще в комментарии бред написан.
```suggestion
//Quality proc. Because the cyanide-syringe is not a child class of the /syringe
/obj/item/weapon/reagent_containers/proc/cause_syringe_fear(mob/living/carbon/human/user)
```